src/sub.fs

```diff
@@ -0,0 +1,88 @@
+namespace Elmish
+
+open System
+
+/// SubId - Subscription ID, alias for string
+type SubId = string
+
+/// Subscription - Generates new messages when running
+type Sub<'msg> = { Start: SubId -> Dispatch<'msg> -> IDisposable }
+
+module Sub =
+
+    module One =
+
+        /// When emitting the message, map to another type
+        let map (f: 'a -> 'msg) (sub: Sub<'a>) : Sub<'msg> =
+            { Start = fun subId dispatch ->
+                sub.Start subId (f >> dispatch) }
+
+    /// None - no subscriptions, also known as `[]`
+    let none : Cmd<'msg> =
+        []
+
+    /// Aggregate multiple subscriptions
+    let batch (subs: #seq<Sub<'msg> list>) : Sub<'msg> list =
```

Comment:
Should have same return type between `none` and `batch` imo.

Reply:
Made some copy pasta there. 🙃 I think you're right about the ID. Especially with what I'm about to post.